pkg/session_recording/recordings.jsx

```diff
@@ -398,6 +491,35 @@
             this.journalctl = null;
         }
 
+        /*
+         * Restarts journalctl.
+         * Assumes journalctl is running.
+         */
+        journalctlRestart() {
+            if(this.journalctlIsRunning()) {
+                this.journalctl.stop();
+                this.recordingMap = {};
+                this.setState({recordingList: []});
+            }
+            this.journalctlStart();
+        }
```

Comment:
The function description and code don't match. First of all, the function doesn't assume journalctl is running, it handles both cases: running and not running. Next, it not only restarts journalctl, it also clears state, and that must be mentioned in the description, if you're going to keep that there.